src/runtime_src/core/tools/common/ReportCmcStatus.cpp

```diff
@@ -34,14 +34,30 @@ ReportCmcStatus::getPropertyTree20202( const xrt_core::device * _pDevice,
                                            boost::property_tree::ptree &_pt) const
 {
   boost::property_tree::ptree pt;
+  pt.put("Description", "CMC");
   try {
-    pt.put("Description", "CMC status");
     pt.put("heartbeat_err_time", xrt_core::device_query<xrt_core::query::heartbeat_err_time>(_pDevice));
     pt.put("heartbeat_count", xrt_core::device_query<xrt_core::query::heartbeat_count>(_pDevice));
     pt.put("heartbeat_err_code", xrt_core::device_query<xrt_core::query::heartbeat_err_code>(_pDevice));
     pt.put("heartbeat_stall", xrt_core::device_query<xrt_core::query::heartbeat_stall>(_pDevice));
     pt.put("status", xrt_core::utils::parse_cmc_status(static_cast<unsigned int>(xrt_core::device_query<xrt_core::query::heartbeat_err_code>(_pDevice))));
-  } catch(const xrt_core::query::no_such_key&) {}
+  }
+  catch(const xrt_core::query::no_such_key&) {}
+  catch(const xrt_core::query::sysfs_error&) {}
+  try {
+    pt.put("scaling_enabled", xrt_core::device_query<xrt_core::query::xmc_scaling_enabled>(_pDevice));
+    pt.put("scaling_support", xrt_core::device_query<xrt_core::query::xmc_scaling_support>(_pDevice));
+    pt.put("scaling_critical_power_threshold", xrt_core::device_query<xrt_core::query::xmc_scaling_critical_pow_threshold>(_pDevice));
+    pt.put("scaling_critical_temp_threshold", xrt_core::device_query<xrt_core::query::xmc_scaling_critical_temp_threshold>(_pDevice));
+    pt.put("scaling_threshold_power_limit", xrt_core::device_query<xrt_core::query::xmc_scaling_threshold_power_limit>(_pDevice));
+    pt.put("scaling_threshold_temp_limit", xrt_core::device_query<xrt_core::query::xmc_scaling_threshold_temp_limit>(_pDevice));
+    pt.put("scaling_power_override_en", xrt_core::device_query<xrt_core::query::xmc_scaling_power_override_enable>(_pDevice));
+    pt.put("scaling_power_override", xrt_core::device_query<xrt_core::query::xmc_scaling_power_override>(_pDevice));
+    pt.put("scaling_temp_override_en", xrt_core::device_query<xrt_core::query::xmc_scaling_temp_override_enable>(_pDevice));
+    pt.put("scaling_temp_override", xrt_core::device_query<xrt_core::query::xmc_scaling_temp_override>(_pDevice));

```

Comment:
This should be more layered:
```
"scaling" : {
     "enabled": "true",
     ...
    "critical_threshold_limits: {
        power_watts: "N/A",
        temp_C = "97"
    }
    ...
}
```

Group them like how you have grouped them while printing. When you make these changes, please add the json output to the PR for review. 
Please add "_watts" or "_C" as well to the node names